consumers/best-practices.md

```diff
@@ -41,6 +41,33 @@ Querying the current price will fail if too much time has elapsed since the last
 The SDKs expose this failure condition in an idiomatic way: for example, the Rust SDK may return `None`, and our Solidity SDK may revert the transaction.
 The SDK provides a sane default for the staleness threshold, but users may configure it to suit their use case.
 
+# Latency
```

Comment:
I wonder if this should go underneath the Confidence Intervals section? I think latency is a concern for a subset of integrators, whereas all integrators need to be aware of CIs.